src/thread.h

```diff
@@ -46,7 +48,17 @@ inline bool StartNewThread(std::thread *thr, const char *name, TFn&& _Fx, TArgs&
 {
 #ifndef NO_THREADS
 	try {
+		static std::mutex cv_m;
```

Comment:
It's a bit weird to suggest the variable has something to do with conditional variables, when it does not. Maybe just call it `mutex` or `thread_start_mutex`?

Reply:
I think both lines of code show what I initially was building :P But I aint doing that till C++20! Fuck that shit ;)